tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -125,79 +125,79 @@
 // Delegate for testing Cortana foreground activation
 // Use text value to guarantee it is the same argument we create
 @interface CortanaVoiceCommandForegroundTestDelegate : NSObject <UIApplicationDelegate>
-@property (nonatomic, readonly) NSMutableDictionary* methodsCalled;
+@property (nonatomic, readonly) StrongId<NSMutableDictionary> methodsCalled;
```

Comment:
Memory leaks from _methodsCalled = [NSMutableDictionary new]; in the old implementation. There was no corresponding release in the dtor. The options were 1. add a dtor 2. use strong id 3. turn on arc. I chose 2 but @DHowett-MSFT  seems less enthused with that choice.

---
In reply to: [82302619](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82302619) [](ancestors = 82302619,82297024,82296469)